features/networking/multus.feature

```diff
@@ -922,7 +920,7 @@ Feature: Multus-CNI related scenarios
     And I run commands on the host:
       | ip link add mvlanp0 type vxlan id 100 remote <%= cb.workr_inf_address %> dev <%= cb.mastr_inf_name %> dstport 14789 |
       | ip link set up mvlanp0                                                                                              |
-      | ip a add 192.168.1.1/24 dev mvlanp0                                                                                 |
+      | ip a add 87.87.8.10/24 dev mvlanp0                                                                                  |
```

Comment:
I think there are other non-routable blocks.

could we use 169.254.0.0/16?, or maybe the TEST-NET-3 reserved 203.0.113.0/24?

https://en.wikipedia.org/wiki/Reserved_IP_addresses

Reply:
198.18.0.0/15 is reserved for Benchmarking, that sounds exciting.  :smiley:

Reply:
Nice. yeah, The bench-marking one really sounds exciting. Re-ran to make sure http://pastebin.test.redhat.com/862033. Also these tests will be soon replaced by multus ```wherabouts``` which will be used to assign dhcp addresses so just matter of short time we need to keep these existing tests running

Reply:
@anuragthehatter Multus whereabouts DHCP is supported in v4.5 and backport will be in v4.4 soon. Compare the way this case using DHCP server, whereabouts is much easy to use, also we do not have formal release docs to describe how to setup the DHCP server as this test case tested, in the future, we may think about to disable this testing if automation fail again.